WebRoot/messages/ZaMsg_ar.properties

```diff
@@ -1008,6 +1008,8 @@ MSG_zimbraPrefMailPollingInterval = \u0627\u0644\u0641\u0627\u0635\u0644 \u0627\
 LBL_asNewMailArrives = \u0639\u0646\u062f \u0648\u0635\u0648\u0644 \u0628\u0631\u064a\u062f \u062c\u062f\u064a\u062f
 LBL_manual = \u064a\u062f\u0648\u064a\u064b\u0627
 LBL_minute = \u062f\u0642\u0627\u0626\u0642
+LBL_recallminute = \u062f\u0642\u0627\u0626\u0642 {0}
```

Comment:
LBL_zimbraFeatureEnableMailRecall and LBL_zimbraFeatureMailRecallInterval missing

Reply:
@silentsakky , we have covered up that in https://github.com/Zimbra/zm-admin-console/pull/146

Reply:
Can we merge both PR? no point in having separate commit/ticket only for translations

Reply:
sure merging https://github.com/Zimbra/zm-admin-console/pull/146 to it

Reply:
merged the code to this branch